flow/keys/save/hex/hex.go

```diff
@@ -40,81 +40,89 @@ type Config struct {
 	Overwrite bool   `flag:"overwrite" info:"bool indicating if we should overwrite an existing config with the same name in the config file"`
 }
 
-var conf Config
+var flags Flags
 
 var Cmd = &cobra.Command{
 	Use:     "hex",
 	Short:   "Save a hex key to the config file",
 	Example: "flow keys save hex --name test --address 8c5303eaa26202d6 --sigalgo ECDSA_secp256k1 --hashalgo SHA2_256 --index 0 --privatekey <HEX_PRIVATEKEY>",
```

Comment:
Hmm, I think we should rethink this command 🤔 

I'm not sure it makes sense as a `keys` command.

Reply:
yes, I agree. Maybe we could that as a part of refactoring commands and writing guidelines so we leave this refactor as purely focused on moving config to the whole project. I had to really break myself to not change some commands because I think there is a lot of good improvements that can be made on the command namings.